docs/flow-visibility.md

```diff
@@ -0,0 +1,45 @@
+#Flow Visibility in Antrea
+
+[Antrea](architecture.md) implements Container Network Interface(CNI) and Network Policy (both Kubernetes and customized)
+ to provide network connectivity and security features for pod workloads. Considering the enormity and dynamism of Kubernetes
```

Comment:
you have extra white spaces at the beginning of most lines in this doc

Reply:
Goland default format. Fixed it.